source/boundary_composition/spherical_constant.cc

```diff
@@ -99,6 +120,9 @@ namespace aspect
       {
         prm.enter_subsection("Spherical constant");
         {
+          prm.declare_entry ("Surface composition", "0",
+                             Patterns::Double (),
+                             "Composition at the surface (lithosphere water/air). Units: none.");
```

Comment:
my proposal would remove this parameter, and add a sentence to the definition of "Outer composition" to say that that is used for the surface when the geometry is a sphere.